### DISCUSSION TITLE
SLOW STARTUP

### DISCUSSION THREAD

**Discussion by gmsfox** [wifiphisher/wifiphisher#1570] [Q&A]
I have a tl-wn722n V2/V3 adapter with all the drivers installed, with the dependencies installed, but when I start wifiphisher the terminal hangs!
 Paralyzing in this timezone

![Captura de Tela (4)](https://user-images.githubusercontent.com/88692724/216746751-2812fb3a-3e2a-4e41-bac7-50896f9e7aa8.png)

**Comment by SOURAV-958** [comment]
Please check out @itz__sourav_edit

**Reply by gmsfox** [reply]
Instagram?

**Comment by FredMagas** [comment]
Did you manage to make it work? I managed to get past this timezone freeze with -nE, but then deauth doesn't work.

**Reply by gmsfox** [reply]
I'm going back now, due to the high workloads! I didn't study anymore. How are you there?
